examples/stable/mysql/README.md

```diff
@@ -10,35 +10,47 @@ This chart bootstraps a single node MySQL deployment on a [Kubernetes](http://ku
 
 - Kubernetes 1.6+ with Beta APIs enabled
 - PV provisioner support in the underlying infrastructure
-- Kanister controller version 0.26.0 installed in your cluster
-- Kanctl CLI installed (https://docs.kanister.io/tooling.html#kanctl)
+- Kanister controller version 0.26.0 installed in your cluster, let's assume in Namespace `<kanister-ns>`
+- Kanctl CLI installed (https://docs.kanister.io/tooling.html#install-the-tools)
 
 ## Installing the Chart
 
-To install the chart with the release name `my-release`:
+To install the MySQL database using the `stable` chart with the release name `mysql-release`:
 
 ```bash
+# Add stable in your local chart repository
 $ helm repo add stable https://kubernetes-charts.storage.googleapis.com/
+
+# Update your local chart repository
 $ helm repo update
 
-$ helm install stable/mysql -n my-release --namespace mysql-test \
+# Install the MySQL database (Helm Version 2)
+$ helm install stable/mysql -n mysql-release --namespace mysql-test \
+    --set mysqlRootPassword='asd#45@mysqlEXAMPLE' \
+    --set persistence.size=10Gi
+
+
+# Install the MySQL database (Helm Version 3)
+$ kubectl create namespace mysql-test
+$ helm install mysql-release stable/mysql --namespace mysql-test \
     --set mysqlRootPassword='asd#45@mysqlEXAMPLE' \
     --set persistence.size=10Gi
+
 ```
 
-The command deploys an instance MySQL in `mysql-test` namespace on Kubernetes cluster with default configurations.
+The command deploys an instance of MySQL in `mysql-test` namespace on Kubernetes cluster with default configurations.
 
 By default a random password will be generated for the root user. For setting your own password, use the `mysqlRootPassword` param as shown above.
 
-You can retrieve your root password by running the following command. Make sure to replace [YOUR_RELEASE_NAME] and [YOUR_NAMESPACE]:
+You can retrieve your root password by running the following command. Make sure to replace [MYSQL_RELEASE_NAME] and [MYSQL_RELEASE_NAMESPACE]:
```

Comment:
Let's remove this and use the same namespace and release name as above ^^

Reply:
```
You can retrieve your root password by running the following command. Make sure to replace [YOUR_RELEASE_NAME] and [YOUR_NAMESPACE]
```
we should go with this, right ?